spec/v2/index.bs

```diff
@@ -1382,6 +1389,46 @@ Example JSON string value:
 ```
 </div>
 
+## Data Type: <dfn>CharacterizationFactorsSourcesSet</dfn> ## {#dt-characterizationfactorssourcesset}
+
+A set of 1 or more <{CharacterizationFactorsSource|Characterization Factors Sources}>.
+
+### JSON Representation ### {#dt-characterizationfactorssourcesset-json}
+
+As an array of objects, with each object conforming to the JSON representation of <{CharacterizationFactorsSource}>.
+
+## Data Type: <dfn element>CharacterizationFactorsSource</dfn> ## {#dt-characterizationfactorssource}
+
+A `CharacterizationFactorsSource` references a source of the characterization factors used in the calculation of the <{CarbonFootprint}> (see [=Pathfinder Framework=] Section 3.2.2).
```

Comment:
Small detail, but would change to "references the source of the characterization factor(s)" as there may be only one characterization factor version used (i.e. AR5), and we are looking for THE source, not A source.